client/README.md

```diff
@@ -11,8 +11,13 @@ This extension provides a Yseop Markup Language (YML) basic support for VS Code.
     * `granule`: Text granule
     * `instance`: Object instance
     * `textu`: Textualization attribute
+![Snippet for textualization Gif.](images/textualization.gif "Snippet for textualization.")
+
 - Code folding
 - KB class attributes completion (needs class to be opened at least once)
+- Basic Yseop Engine model based type completion
```

Comment:
Copy whatever you write where I placed my first comment, I guess.

Reply:
Ok, I'll do the changes.